apt.txt

```diff
@@ -0,0 +1,2 @@
+libgl1-mesa-glx
```

Comment:
Add a comment here that these are the packages for binder that are needed to run the headless display?

Reply:
Added to file.